ar4_common/package.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>ar4_common</name>
+  <version>0.0.1</version>
+  <description>
+    Launch Gazebo simulation with Ar4 robot.
```

Comment:
Is this description correct for the `ar4_common` package?

Reply:
Fixed.